create_infotexts.py

```diff
@@ -333,42 +344,49 @@ def select_best_mapping_for_depicted_person(self, flipped_name):
+        :param place_string: string value <Ort, foto> in metadata item.
+        :param places_mapping: Dictionary containing Commons:Medelhavsmuseet/batchUploads/Cypern_places kw mappings
         :return: None (output stored in object attribute
         """
-        places_mapping = load_places_mapping()
-        # print(places_mapping)
-
-        name_as_wikitext = ""
-
-        if place_string == "":
+        place_as_wikitext = ""
+        if not place_string:
             self.data["depicted_place"] = ""
             return
 
-        elif place_string in places_mapping:
-            if not places_mapping[item["Ort, foto"]]["wikidata"] == "-":
-                self.data["depicted_place"] = "{{city|1={wikidata}|link=wikidata}}".format(
-                    wikidata=places_mapping[item["Ort, foto"]]["wikidata"][2:]
+        if place_string in places_mapping:
+            if not places_mapping[place_string]["wikidata"] == "-":
+                 place_as_wikitext = "{{city|1={wikidata}}}".format(
+                    wikidata=places_mapping[place_string]["wikidata"][2:]
                 )
-            else:
-                self.data["depicted_place"] = "{{city|1={commons}|link=commons}}".format(
-                    commons=places_mapping[item["Ort, foto"]]["commons"]
+            elif place_string == "Stockholm":  # Mainly interiors from buildings gardens
+                self.meta_cats.append("[[Media_contributed_by_SMVK_taken_in_Stockholm]]")
+                self.data["depicted_place"] = ""
+
+            elif place_string == "Macheras":  # No WP article, highly ambiguous. Might refer to "Machairas Monestary"
+                self.meta_cats.append("Media_contributed_by_SMVK_possibly_depicting_Machairas_Monastary")
+                self.data["depicted_place"] = ""
+
+
+            elif not places_mapping[place_string]["commons"] == "-":
+                place_as_wikitext = "{{city|1={commons}}".format(
+                    commons=places_mapping[place_string]["commons"]
                 )
 
-        elif place_string == "Stockholm": # Mainly interiors from buildings gardens
-            self.content_cats.append("[[Media_contributed_by_SMVK_2017-02_taken_in_Stockholm]]")
-            self.data["depicted_place"] = ""
+            else:
+                self.meta_cats.append("Media_contributed_by_SMVK_without_depicted_place_value")
+                self.data["depicted_place"] = ""
+                return
 
-        elif place_string == "Macheras": # No WP article, highly ambiguous
-            self.data["depicted_place"] = ""
-        else:
-            raise ValueError("ValueError: Not able to map <Ort, foto>: {}".format(place_string))
+        # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
+        if "commons" in places_mapping[place_string].keys() and places_mapping[place_string]["commons"] != "-":
+            self.content_cats.append(places_mapping[place_string]["commons"])
 
```

Comment:
you need an `else` here (to the `if place_string in places_mapping`) which also sets 
```python
self.meta_cats.append("Media_contributed_by_SMVK_without_depicted_place_value")
place_as_wikitext = place_string
```